appcues/src/main/java/com/appcues/statemachine/effects/ExperienceActionEffect.kt

```diff
@@ -8,7 +8,7 @@ import com.appcues.statemachine.SideEffect
 internal data class ExperienceActionEffect(private val actions: List<ExperienceAction>) : SideEffect {
 
     override suspend fun launch(processor: ActionProcessor): Action? {
-        processor.process(actions)
+        processor.processAsync(actions)
```

Comment:
maybe just add a comment here for our future selves noting (1) this effect is only used for processing the post-flow-completion actions AFTER the experience has transitioned to Idle and (2) they must kick off their own new coroutine so that the state machine can complete the current transition, unblocked, and return back to the resting/idling state and release the mutex held on the current action processing instance (the current flow completion)